Backpack/Badge/Classes/Badge.swift

```diff
@@ -0,0 +1,111 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2022 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import UIKit
+
+@objc
+public class BPKBadge: UIView {
+    private let label = BPKLabel(fontStyle: .textXsEmphasized)
+    
+    public var message: String? {
+        get { label.text }
+        set { label.text = newValue }
+    }
+    
+    public var type: BPKBadgeType = .success {
+        didSet { updateLookAndFeel() }
+    }
+    
+    public convenience init(type: BPKBadgeType = .success, message: String) {
+        self.init(frame: .zero)
+        self.message = message
+        self.type = type
+        updateLookAndFeel()
+    }
+    
+    public override init(frame: CGRect) {
+        super.init(frame: frame)
+        setup()
+    }
```

Comment:
In the original version, the setup() function would default to the `.success` type. We should be careful not to break this flow if implementers assumed this default behaviour

Reply:
This is now done in the initialisation of the type property! `.success` will be the default